docs/guides/faq.md

```diff
@@ -0,0 +1,91 @@
+## FAQ
+
+**Q: What is video.js?**
+> video.js is an extendable wrapper around the native video element. It does the following:
+>  * Offers a plugin API so that different types of video can be handed to the native video element. IE:  HLS, Flash, or HTML5 video.
+>  * Polyfills and unifies the native video api across browsers
```

Comment:
It doesn't usually polyfill anything unless completely broken. It generally sits on top to make a unified version.